src/components/EventRegistrantsModal/EventRegistrantsModal.tsx

```
@@ -142,34 +181,50 @@ export const EventRegistrantsModal = (props: ModalPropType): JSX.Element => {
+                  <p className="me-2">No Registrations found</p>
+                  <a
+                    className="underline"
+                    href="#"
+                    onClick={() => {
+                      setOpen(true);
+                    }}
+                  >
+                    Add Onspot Registration
+                  </a>
+                </div>
+              </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)